go.mod

```diff
@@ -6,24 +6,25 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gonum/graph v0.0.0-20170401004347-50b27dea7ebb
-	github.com/google/go-cmp v0.4.0
-	github.com/openshift/api v0.0.0-20200901182017-7ac89ba6b971
-	github.com/openshift/build-machinery-go v0.0.0-20200819073603-48aa266c95f7
-	github.com/openshift/client-go v0.0.0-20200827190008-3062137373b5
+	github.com/google/go-cmp v0.5.2
+	github.com/openshift/api v0.0.0-20201214114959-164a2fb63b5f
+	github.com/openshift/build-machinery-go v0.0.0-20200917070002-f171684f77ab
+	github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49
 	github.com/openshift/library-go v0.0.0-20200910214143-887092e305c1
 	github.com/prometheus/common v0.10.0
-	github.com/spf13/cobra v1.0.0
+	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/apiserver v0.19.0
-	k8s.io/client-go v0.19.0
-	k8s.io/component-base v0.19.0
-	k8s.io/klog/v2 v2.3.0
+	github.com/stretchr/testify v1.6.1
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/apiserver v0.20.0
+	k8s.io/client-go v0.20.0
+	k8s.io/component-base v0.20.0
+	k8s.io/klog/v2 v2.4.0
 )
 
 replace (
+	github.com/openshift/library-go => github.com/Danil-Grigorev/library-go v0.0.0-20210217125431-c0ce7f525449
```

Comment:
this doesn't seem proper, shouldn't we have this pointing to the openshift/library-go after some patch has merged?

Reply:
we will bump the library here when https://github.com/openshift/library-go/pull/895 is merged